cdc/kv/client.go

```diff
@@ -390,7 +396,11 @@ MainLoop:
 			if !ok {
 				stream, err = c.getStream(ctx, rpcCtx.Addr)
 				if err != nil {
-					return errors.Trace(err)
+					// if get stream failed, maybe the store is down permanently, we should try to relocate the active store
+					log.Warn("get grpc stream client failed", zap.Error(err))
+					bo := tikv.NewBackoffer(ctx, tikvRequestMaxBackoff)
+					c.regionCache.OnSendFail(bo, rpcCtx, needReloadRegion(sri.failStoreIDs, rpcCtx), err)
+					continue MainLoop
```

Comment:
if tikv is not active, will we fall into an endless loop?

Reply:
`c.regionCache.OnSendFail(bo, rpcCtx, needReloadRegion(sri.failStoreIDs, rpcCtx), err)` has set this tikv as failed store

Reply:
What happens if the server is all down?

Reply:
cdc retries to connect tikv until TiKV up, and the cluster is avialable

Reply:
In fact this error only happens when gRPC connection is not established, which means at the CDC startup procedure.